orca-queue/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
I think this can disappear now.

Reply:
no, it is still needed, as `spy(DefaultRegistry())` is used in the tests, and DefaultRegistry is also a final class

Reply:
Aaah, ok.